manifests/boolean.pp

```diff
@@ -36,8 +36,11 @@
     default              => undef,
   }
 
-  selboolean { $name:
-    value      => $value,
-    persistent => $persistent,
+  # Do nothing unless SELinux is enabled
+  if $facts['selinux'] == true {
```

Comment:
The `== true` isn't really needed.

```suggestion
  if $facts['selinux'] {
```

Reply:
@ekohl I can change those but I've actually ran into cases where it *is* needed.  On some nodes just saying "if $foo" will return as true as long as the value is *defined*, even when it is false.  IME explicitly requiring a variable to be true results in more predictable behavior.

Reply:
@ekohl I've updated the manifest files to pass rubocop tests.